src/ConnectionInfo/ConnectionInfo.jsx

```diff
@@ -76,7 +76,7 @@ class ConnectionInfo extends React.Component {
             <Group direction="row">
               <Icon icon="keypad" />
               <Text color="lightGrey" size={14}>
-                Connect to a reader
+                No reader connected
```

Comment:
this here was confusing when using a screenreader. "Connect to a reader" is the heading text on the next element on the page, so the same message was repeated twice. "No reader connected" is a little more contextual / is the compliment to the alternative connected reader state in my opinion but let me know what you think!